src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
```

Comment:
The stateless component Pagination is missing the perPageSelector with the options 3, 5, 10, 20 to change the perPage.